js/renovation/ui/editors/check_box/__tests__/check_box.test.tsx

```diff
@@ -224,6 +224,20 @@ describe('CheckBox', () => {
             checkBox.onWidgetClick({} as Event);
             expect(checkBox.props.value).toBe(false);
           });
+
+          each([
+            { initial: true, expected: false },
+            { initial: false, expected: null },
+            { initial: undefined, expected: true },
```

Comment:
value cannot be `undefined` on this level of code.
```suggestion
            { initial: undefined, expected: true },
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/22616/commits/1cbf6a879183845bb9906f1e66ad9e0e563979a4